{"name": "Register user with incorrect auth data: no_name", "status": "passed", "steps": [{"name": "Register user", "status": "passed", "start": 1724244964220, "stop": 1724244964505}, {"name": "Delete user", "status": "passed", "start": 1724244964505, "stop": 1724244964505}], "attachments": [{"name": "stdout", "source": "844b8410-131f-4d42-be09-9c9c719065dd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "auth_params", "value": "'no_name'"}], "start": 1724244964220, "stop": 1724244964506, "uuid": "db6e2861-5697-4c90-b848-c2bed0982a86", "historyId": "d104db2eb8af389d633523ecacebd949", "testCaseId": "7d3802100fc42b489225ca410ffa4c93", "fullName": "tests.test_auth.TestAuthRegister#test_register_incomplete_data_unsuccessful", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestAuthRegister"}, {"name": "host", "value": "opeshko"}, {"name": "thread", "value": "43448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}